core/requires.js

```diff
@@ -88,5 +88,6 @@ goog.require('Blockly.geras.Renderer');
 // Classic is the default theme.
 goog.require('Blockly.Themes.Classic');
 goog.require('Blockly.Themes.Dark');
+goog.require('Blockly.Themes.Deuteranopia');
 goog.require('Blockly.Themes.HighContrast');
-goog.require('Blockly.Themes.Modern');
```

Comment:
I would comment it out instead, similar to requires that aren't included in the rest of this file.

Reply:
done